StripePaymentsUI/StripePaymentsUI/Source/Helpers/String+Localized.swift

```diff
@@ -76,12 +76,9 @@ extension String.Localized {
     @_spi(STP) public static var cvc_section_title: String {
         STPLocalizedString(
             "Confirm your %@",
-            "Section title for entering your CVC/CVV. e.g. 'Confirm your CVC' or 'Confirm your CVV'"
+            "Section title for entering your CVC. e.g. 'Confirm your CVC'"
         )
     }
```

Comment:
Seems like we can get rid of `cvc_section_title`, and create a new string that doesn't take in a variable.

Reply:
++ to leaving as is, otherwise we need to get new strings translated.